.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -24,66 +24,62 @@
 
 <!-- Please add direct links where your modifications can be seen in the documentation -->
 
-* https://deploy-preview-{your pr number}--boosted.netlify.app/
+* https://deploy-preview-{your_pr_number}--boosted.netlify.app/
 
 ### Checklist
 
 <!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
 <!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
 
+#### Contribution
+
 - [ ] I have read the [contributing guidelines](https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/blob/main/.github/CONTRIBUTING.md)
+
+#### Design
+
+- [ ] My change respects the design guidelines
+- [ ] My change is compatible with responsive display
+
+#### Development
+
 - [ ] My code follows the code style of the project _(using `npm run lint`)_
-- [ ] My change introduces changes to the documentation
-- [ ] My change introduces changes to the migration guide
-- [ ] I have updated the documentation accordingly
-- [ ] I have added tests to cover my changes
+- [ ] I have added JavaScript unit tests to cover my changes
 - [ ] All new and existing tests passed
```

Comment:
Is it necessary since it is launched automatically by GitHub actions?

Reply:
We use this trick with the linter (that is launched with CI as well I think)

Reply:
Everything is launched automatically by GitHub Actions (CI). Everything (except CSpell) can be launched manually on one branch. So you want to recap this information here as well?
Vote with your thumbs up/down on this comment.

Reply:
When beginning on the project, it's not always easy to know which npm commands to run before pushing our commits. So, I think it could be interesting to:
- add this "using npm run command_name" 
- OR avoid looking/hesitating each time for the command to use
- OR to add a more generic sentence reminding to run the commands from the package.json file before pushing our commits
Your opinion ?

Reply:
These informations are certainly useful but I don't think that the PR template is the right place.
Maybe in a 'How to contribute' guide?

Reply:
Maybe something like `all Github actions validated` ? (and remove the linter and js-test part)

Reply:
I agree that's very boring to check "My code follows the code style" since the CI already flags things out. I try to remove this line and remove as well "All new and existing tests passed" because it's kind of obvious that if the CI is red, it's for something. No need to maintain checkboxes with something already handling it automatically IMO. Same thing as the other comment. I'm resolving it but feel free to reopen it if you disagree or have other comment.